python/modules/IcePy/Communicator.cpp

```diff
@@ -983,25 +983,25 @@ communicatorFindAllAdminFacets(CommunicatorObject* self, PyObject* /*args*/)
     PyTypeObject* objectType = reinterpret_cast<PyTypeObject*>(lookupType("Ice.Object"));
     PyObjectHandle plainObject{objectType->tp_alloc(objectType, 0)};
```

Comment:
The usage of plainObject is puzzling, if the facet was not implementing in Python we just return this plainObject which is of no use. We should consider returning None to indicate a native implemented facet.